modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -190,12 +191,10 @@ export function flushCoinsData(): string {
 /**
  * Returns the create forwarder method calling data
  *
- * @param {string} forwarderVersion - forwarder version for the address to be initialized
  * @returns {string} - the createForwarder method encoded
  */
-export function getAddressInitializationData(forwarderVersion?: number): string {
-  const impliedForwarderVersion = forwarderVersion ?? defaultForwarderVersion;
-  return impliedForwarderVersion === defaultForwarderVersion ? createForwarderMethodId : v1CreateForwarderMethodId;
+export function getAddressInitializationData(): string {
+  return createForwarderMethodId;
```

Comment:
@mmcshinsky-bitgo I am wondering if removing an optional param is a breaking change?

Reply:
Only required parameters are breaking changes, but we should clearly detail the change in the commit description.

Reply:
Got it. Will update the description

Reply:
Does removing the optional parameter create situations where the end consumers will need to change their own code to get previous expectations that the function provided in the expected logic flow?

Reply:
Actually to give a little background on the change, I had added this optional parameter in this [PR](https://github.com/BitGo/BitGoJS/pull/2515). But now I have made the function just like it was before this mentioned PR by removing the optional parameter. 

The method will behave in the same manner for the users who were not passing the optional parameter or were passing the default value of optional parameter. But if a non-default value was passed to the method, then it returned a different response previously, and now there are no 2 different responses. It is just a single response (i.e. for the default value).